strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/providers/layout/Maven2LayoutProvider.java

```diff
@@ -58,7 +58,7 @@ protected MavenArtifactCoordinates getArtifactCoordinates(RepositoryPath reposit
     {
         MavenArtifact artifact = MavenArtifactUtils.convertPathToArtifact(repositoryPath);
 
-        return new MavenArtifactCoordinates(artifact);
+        return artifact != null ? new MavenArtifactCoordinates(artifact) : null;
```

Comment:
@fuss86 it seems that previous implementation not assumed to return `null`, is there any reasons to do it now?

Reply:
previous implementation created `Artifact` even if provided `RepositoryPath` couldn't have been an `Artifact`

Reply:
yeah, I mean that we probably better to throw an exception here instead of return `null`

Reply:
👍

Reply:
I got rid of this `null` check, it wasn't necessary, thanks.